docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -160,10 +162,39 @@ The following configuration properties are available:
        This flag allows ignoring the value provided in the metadata document.
        Default is ``true``.
 
+.. _trino-oauth2-refresh-tokens:
+
 Refresh tokens
 ^^^^^^^^^^^^^^
 
-In order to start using refresh tokens flows, the following property must be
+*Refresh tokens* allow you to securely control the length of user sessions
+within applications. The refresh token has a longer lifespan (ttl) and is used
+to refresh the *access token* that has a shorter lifespan. With refresh tokens
+behind the scenes, users can remain logged in for an extended duration without
+interruption by another login request.
```

Comment:
The "with refresh tokens behind the scenes" still does not read well. a) It's colloquial, and b) it's not user-centric, and the sentence is awkward in general.

Something like "When these tokens are used together, user sessions are interrupted less frequently by login requests."

Reply:
This sentence retains the content and feedback from the SMEs without a bloggy tone and has parallel sentence construction:

> When refresh tokens
> are used in conjunction with access tokens, users can remain logged in for an
> extended duration without interruption by another login request.

@m57lyra thanks for your review (always room for improvement), and please review again.